src/pybind/mgr/dashboard/frontend/src/app/core/layouts/login-layout/login-layout.component.scss

```diff
@@ -29,4 +29,19 @@
     min-height: 100vh;
     width: 100vw;
   }
+
+  .form-inline {
+    a {
+      color: bd.$white;
+    }
+
+    a:hover {
+      color: bd.$gray-500;
+    }
+  }
```

Comment:
I was suggesting to @aaSharma14 in another PR to name color vars after its purpose rather than textual color descriptions... For example, `bd.$white` could better be something like `bd.$text-color-over-dark-background` or `bd.$foreground-color-over-dark-background` (added to boostrap-defaults SCSS).

Reply:
Just a concern over there. The _bootstrap-defaults.scss looks cleaner now. So if we add variables like this for each component, wouldnt it make a mess later? (like we had previously) @epuertat

Reply:
Well, that's the point with 'semantic' (or whatever this is called) variables. If you define something like `fg-color-over-dark-bg` (or `high-contrast-text-color`), that may apply to text in every other component using a dark background. If you need to define many of such variables per component, probably it's because the side-wide theming is being ignored.

Reply:
Make sense. Updated the PR.